CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 ## Change Log
 
+### v1.1.0
+
+* Clean up repo documentation for going public.
+* New API k4a_device_get_color_control_capabilities() to read color control capabilities was added. (#319)
+* C++ wrapper added.
+* Linux, Color camera support added.
+* K4A_COLOR_CONTROL_AUTO_EXPOSURE_PRIORITY deprecated. (#277)
+* New K4A_*_TRACK tags added to recordings for track identification. (#259)
```

Comment:
We should escape this * so that we don't italicized everything else after it.
 ```suggestion
* New K4A_\*_TRACK tags added to recordings for track identification. (#259)
```